client/me/help/gm-closure-notice/index.jsx

```diff
@@ -0,0 +1,176 @@
+			guessedTimezone
+		),
+		moment
+			.tz(
+				isBusinessOrEcommercePlan ? businessAndEcommerceReopensAt : defaultReopensAt,
+				guessedTimezone
+			)
+			.subtract( 1, 'days' ),
+		moment.tz(
+			isBusinessOrEcommercePlan ? businessAndEcommerceReopensAt : defaultReopensAt,
+			guessedTimezone
+		),
+	];
+
+	if ( ! currentDate.isBetween( displayAt, reopensAt ) ) {
+		return null;
+	}
+
+	const HEADING = translate( 'Limited Support %(closesAt)s – %(closedUntil)s', {
+		args: {
+			closesAt: closesAt.format( DATE_FORMAT_SHORT ),
+			closedUntil: closedUntil.format(
+				closedUntil.isSame( closesAt, 'month' ) ? 'D' : DATE_FORMAT_SHORT
+			),
+		},
+	} );
+
+	const mainMessageArgs = {
+		closesAt: closesAt.format( DATE_FORMAT_LONG ),
+		closedUntil: closedUntil.format( DATE_FORMAT_LONG ),
+		reopensAt: reopensAt.format( DATE_FORMAT_LONG ),
+	};
+
+	const MAIN_MESSAGES = {
+		before: {
+			hasPlan: translate(
+				'Live chat support will be closed %(closesAt)s through %(closedUntil)s, included. Email support will be open during that time, and we will reopen live chat on %(reopensAt)s.',
+				{ args: mainMessageArgs }
+			),
+			nonPlan: translate(
+				'Private support will be closed %(closesAt)s through %(closedUntil)s, included. We will reopen private support on %(reopensAt)s.',
+				{ args: mainMessageArgs }
+			),
+		},
+		during: {
+			hasPlan: translate(
+				'Live chat support will be closed through %(closedUntil)s, included. Email support will be open during that time, and we will reopen live chat on %(reopensAt)s.',
+				{ args: mainMessageArgs }
+			),
+			nonPlan: translate(
+				'Private support will be closed through %(closedUntil)s, included. We will reopen private support on %(reopensAt)s.',
+				{ args: mainMessageArgs }
+			),
+		},
+	};
+
+	const REASON_MESSAGES = {
+		hasPlan: translate(
+			'Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together to work on improving our services, building new features, and learning how to better serve our customers like you. But never fear! If you need help in the meantime, you can submit an email ticket through the contact form: {{contactLink}}https://wordpress.com/help/contact{{/contactLink}}',
+			{
+				components: {
+					contactLink: <a href="/help/contact" />,
+				},
+			}
+		),
+		nonPlan: translate(
+			'Why? Once a year, the WordPress.com Happiness Engineers and the rest of the WordPress.com family get together to work on improving our services, building new features, and learning how to better serve our customers like you. But never fear! If you need help in the meantime, check our support site at {{supportLink}}https://en.support.wordpress.com{{/supportLink}}',
+			{
+				components: {
+					supportLink: <a href="https://en.support.wordpress.com" />,
+				},
+			}
+		),
+	};
+
+	const FORUMS_NOTE = translate(
+		'Our staff will be keeping an eye on the {{forumLink}}Forums{{/forumLink}} for urgent matters.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Our staff will be keeping an eye on the {{link}}Forums{{/link}} for urgent matters.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).